docs/Configuration.md

```diff
@@ -1614,6 +1614,12 @@ const config: Config = {
 export default config;
 ```
 
+### `showSeed` \[boolean]
+
+Default: `false`
```

Comment:
Such a great option, I love it 💕

Reply:
I was just about to tag you 🙂 Does this fit your use case and usage?

Reply:
I'm thinking you can just document setting this config flag instead of printing the seed yourself

Reply:
It will definitely help a lot. So far `@fast-check/jest ` was mostly responsible for building a seed for `fast-check` and appending it to the name of the `it` (and also provide a `it` tailored for pbt). With both `seed` and `showSeed` options altering the name of the test might not be that required.

Reply:
I haven't documented the removal of the seed printing yet, but that was my idea when I opened https://github.com/dubzzz/fast-check/issues/3287